be/src/exprs/utility_functions.cpp

```diff
@@ -39,4 +39,82 @@ BooleanVal UtilityFunctions::sleep(FunctionContext* ctx, const IntVal& seconds)
     return BooleanVal(true);
 }
 
+static void convert_to_string(doris_udf::StringVal& src, std::string& str_dst) {
+    for (int i = 0; i < src.len; i++) {
+        str_dst += src.ptr[i];
+    }
+    return;
+}
+
+static BooleanVal split_string(std::string& str_src, std::vector<std::string>& str_vec_dst) {
```

Comment:
there is already have split in the util, refer to be/src/util/string_util.h or gutils

Reply:
I will re-check this split function.